src/components/pinAnimatedInput/views/pinAnimatedInputView.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable react/no-array-index-key */
 import React, { Component } from 'react';
 import { View } from 'react-native';
-import Animated, { Easing } from 'react-native-reanimated';
+import * as Animatable from 'react-native-animatable';
```

Comment:
better use `import { View as AnimatableView } from 'react-native-animatable'` import * .... is almost always very expensive import